### PR TITLE
`conditional` & `entity-filter`: add ability to filter through `entity_id` & add `entity-filter` `conditional`'s conditions

### PR DESCRIPTION
**Pull request by Quentame**
## Proposed change

- ~refactor: Use same codebase for `conditional` card, `entity-filter` card & `entity-filter` badge~
- add to `conditional` card: `state` condition handle `entity_id`
- add to `conditional` card: `numeric_state` condition handle `entity_id`
- ~add to `entity-filter` card & badge: (legacy) "`operator`" condition handle `entity_id`~
- add to `entity-filter` card & badge: ALL `conditional` card conditions: `state`, `numeric_state`, `screen`, `user`, `or`, `and`
- fix to `entity-filter` card & badge: "`operator`" condition `in` with number
- fix to `entity-filter` card: error when no entities
- NO BREAKING CHANGES as legacy `entity-filter` config is still supported !

### entity-filter card

| Current dev  | NEW |
| ------------- | ------------- |
| entity-filter_dev  | entity-filter_new |

### conditional card

| Current dev  | NEW |
| ------------- | ------------- |
| conditional_dev  | conditional_new |

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/19182
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by Quentame — COMMENTED** [review]
Need to be updated with new parents PR changes

File: source/_dashboards/conditional.markdown [hunk 1]
```diff
@@ -188,3 +190,53 @@ users:
   description: User ID that can see the card (unique hex value found on the Users configuration page).
   type: list
 {% endconfiguration %}
+
+### And
+
+Specify that both conditions should be met.
```
Comment (c0ffeeca7) [inline source/_dashboards/conditional.markdown]:
```suggestion
Specify that both conditions must be met.
```

File: source/_dashboards/conditional.markdown [hunk 2]
```diff
@@ -188,3 +190,53 @@ users:
...
+  type: string
+conditions:
+  required: false
+  description: List of conditions to check. See [available conditions](#conditions-options).
+  type: list
+{% endconfiguration %}
+
+### Or
+
+Specify that at least one of conditions should be met.
```
Comment (c0ffeeca7) [inline source/_dashboards/conditional.markdown]:
```suggestion
Specify that at least one of the conditions must be met.
```

File: source/dashboards/badges.markdown [hunk 3]
```diff
@@ -93,34 +99,227 @@ image:
...
+  type: string
+conditions:
+  required: false
+  description: List of conditions to check. See [available conditions](#conditions-options).
+  type: list
+{% endconfiguration %}
+
+### Or
+
+Specify that at least one of conditions should be met.
```
Comment (c0ffeeca7) [inline source/dashboards/badges.markdown]:
```suggestion
Specify that at least one of the conditions must be met.
```

File: source/_dashboards/entity-filter.markdown [hunk 4]
```diff
@@ -74,32 +81,225 @@ format:
...
+  type: string
+users:
+  required: true
+  description: User ID that can see the entity (unique hex value found on the Users configuration page).
+  type: list
+{% endconfiguration %}
+
+### And
+
+Specify that both conditions should be met.
```
Comment (c0ffeeca7) [inline source/_dashboards/entity-filter.markdown]:
```suggestion
Specify that both conditions must be met.
```

File: source/_dashboards/entity-filter.markdown [hunk 5]
```diff
@@ -74,32 +81,225 @@ format:
...
+  type: string
+conditions:
+  required: false
+  description: List of conditions to check. See [available conditions](#conditions-options).
+  type: list
+{% endconfiguration %}
+
+### Or
+
+Specify that at least one of conditions should be met.
```
Comment (c0ffeeca7) [inline source/_dashboards/entity-filter.markdown]:
```suggestion
Specify that at least one of the conditions must be met.
```

File: source/dashboards/badges.markdown [hunk 6]
```diff
@@ -93,34 +99,227 @@ image:
...
+users:
+  required: true
+  description: User ID that can see the entity (unique hex value found on the Users configuration page).
+  type: list
 {% endconfiguration %}
 
-### Examples
+### And
+
+Specify that both conditions should be met.
```
Comment (c0ffeeca7) [inline source/dashboards/badges.markdown]:
```suggestion
Specify that both conditions must be met.
```

**Comment by c0ffeeca7** [comment]
tiny style tweaks. For another time (at the risk that I bore you with this information): If you enable the option that maintainers can modify the PR, I can push nitpicks like that myself.

**Comment by c0ffeeca7** [comment]
@Quentame, Not sure if you've seen the nitpick review comments yet. If you push the tiny tweaks that were suggested, this can make it into beta. I don't have the powers to do that.

**Review by c0ffeeca7 — APPROVED** [review]
Nice! Thank you, @Quentame :+1:

**Pull request merged**